environment/tokens.rst

```diff
@@ -0,0 +1,99 @@
+#######################
+Science Platform Tokens
+#######################
+
+The Notebook Aspect of the Rubin Observatory Science Platform provides access tokens for interacting with other services provided by the Science Platform.
+An example is the TAP service provided by the API Aspect.
+When interacting with authenticated services for within the Notebook Aspect you typically doesn't need to interact with the access tokens since the various clients are able to pick up the token from your user environment.
+A feature of the Notebook Aspect is that this access token can also be used outside the Notebook Aspect to e.g. access the TAP service from TOPCAT on a local machine.
+
+This page will give you an introduction to the access token, where to find it, how to inspect it, and how to use it in an environment outside the Notebook Aspect.
+Note that the access token has a finite lifetime and is refreshed on each login to the Notebook Aspect.
+
+The access token
+================
+
+The access token is mounted in a known location in your home space.
+To view it, you can simply look a the special file called ``~/.access_token``.
+From within a `terminal`_ execute the following command.
+
+.. code-block:: bash
+
+   cat ~/.access_token
+
+.. _`terminal`: https://jupyterlab.readthedocs.io/en/latest/user/terminal.html
+
+Provided with the environment is a notebook that allows you to inspect aspects of the token using the ``analyze`` endpoint of the authentication service.
+By default, this notebook is found at ``~/notebooks/notebook-demo/token-info.ipynb``.
```

Comment:
Maybe simplify this because the `/analyze` endpoint might not be immediately relevant to readers.

```suggestion
The environment includes a notebook, at ``~/notebooks/notebook-demo/token-info.ipynb``, that allows you to inspect a token.
```